blivet/osinstall.py

```diff
@@ -1465,176 +1470,22 @@ def _update_custom_storage_ksdata(self):
             parent = getattr(self.ksdata, list_attr)
             parent.dataList().append(data)
 
-        self.fsset = FSSet(self.devicetree)
+        self.config = StorageDiscoveryConfig()
+        self.autopart_type = AUTOPART_TYPE_LVM
 
-    def reset(self, cleanup_only=False):
-        Blivet.reset(self, cleanup_only)
+        self.__luks_devs = {}
 
+        # these will both be empty until our reset method gets called
+        # instantiate our own devicetree here to override the default created
+        # in Blivet so that protected_dev_specs gets handled
+        self.devicetree = DeviceTree(passphrase=self.encryption_passphrase,
+                                     luks_dict=self.__luks_devs,
+                                     ignored_disks=self.ignored_disks,
+                                     exclusive_disks=self.exclusive_disks,
+                                     disk_images=self.disk_images,
+                                     protected_dev_specs=self.config.protected_dev_specs)
```

Comment:
Right; now that I will be able to test again, I'll do the protected device handling changes and push those with my next PR. Same with the super().reset call.